python/ray/_private/utils.py

```diff
@@ -1967,6 +1967,11 @@ def pasre_pg_formatted_resources_to_original(
             continue
         original_resources[key] = value
 
+    # kBundle_ResourceLabel should be invisible to the user.
+    BUNDLE_RESOURCE_LABEL = "bundle"
+    if BUNDLE_RESOURCE_LABEL in original_resources:
+        original_resources.pop(BUNDLE_RESOURCE_LABEL)
+
```

Comment:
I think this has two issues:
1. it will break a custom resource call `my_bundle`.
2. It doesn't apply to other languages.

Reply:
For 1., it is pre-existing here: https://github.com/ray-project/ray/blob/master/src/ray/core_worker/core_worker.cc#L2123